meetings/2020_11_24/README.md

```diff
@@ -0,0 +1,110 @@
+# ROS 2 Security Working Group Meeting Minutes
+24 Nov 2020
+
+[Meeting Recording](https://youtu.be/7ZJidRtTqXI) | [Meeting Announcement](https://discourse.ros.org/t/security-wg-meeting/17519)
+
+
+## Agenda
+
+- Administrivia: future meeting minutes
+- G-turtle goals
+  - MoveIt2 security use case
+  - ROS 2 without a file system, [rcl #545](https://github.com/ros2/rcl/issues/545) and [discourse post](https://discourse.ros.org/t/ros-2-without-a-file-system/16942)
+  - [Galactic Roadmap](https://index.ros.org/doc/ros2/Roadmap/#id2) 
+  - sros2 quality status: any comments?
+- ROS2 secure launch and access control
+- [RMF](https://osrf.github.io/ros2multirobotbook) as a use case, see the [demo](https://github.com/osrf/rmf_demos)
+- Revoking keys
+
+## Attendees
+[Iker Luengo Gil](https://github.com/IkerLuengo),
+[Jacob Hassold](https://github.com/jhdcs),
+[Jaime Martin Losa](https://github.com/JaimeMartin),
+[Jeremie Deray](https://github.com/artivis),
+[Kyle Fazzari](https://github.com/kyrofa),
+[Marco Gutierrez](https://github.com/marcoag),
+[Mikael Arguedas](https://github.com/mikaelarguedas),
+[Ruffin White](https://github.com/ruffsl),
+[Sid Faber](https://github.com/sidfaber)
+
+
+## Administrivia
+
+Following a brief discussion, it was decided to move new meeting minutes to the [ros-security/github repo](https://github.com/ros-security/community). Existing meeting minutes in the [ROS wiki](http://wiki.ros.org/ROS2/WorkingGroups/Security) will not be ported.
+
+The [vulnerability remediation procedure PR](https://github.com/ros-security/community/pull/8) is still open for comments.
+
+
+## G-Turtle goals
+
+Five open items could become part of our G-Turtle deliverables:
+
+### Reference implementation with MoveIt
+
+Goal would be to demonstrate "Hey, look, here's an example of a real system that's secured." Although the config may be able to stand on its own, it would be more useful as an example.
+This example will also be useful for us to find issues with the security implementation on a complex system to test: CPU / network utilization, what to sign, what to encrypt, overall impact to the system.
+This also becomes a proving ground for NoDL.
+
+Simulation may be challenging; a simulated implementation may not quite match the real world implementation. However, we should be able to spec the project in stages. Start simple and build upon the demo.
+
+### Enable DDS security without a file system
+
+The scope of this issue is much wider than just security. Success depends upon buy-in from both the micro-ROS community and from Open Robotics.
+
+The agrees to continue to move the discussion forward to flesh out a design, but not to perform any work on the code at this time.
```

Comment:
```suggestion
The WG agrees to continue to move the discussion forward to flesh out a design, but not to perform any work on the code at this time.
```

Reply:
Updated in a6e2e0e927a898d850106dbe5e7725534954701b